spec/message.abnf

```diff
@@ -1,18 +1,22 @@
-message = pattern / complex-message
+message = simple-message / complex-message
 
-complex-message = "{{" [s] *(declaration [s]) body [s] "}}"
+simple-message = [simple-start pattern]
+simple-start = text-start / text-escape / expression
 
-declaration = input-declaration / local-declaration
+pattern = *(text / expression)
+
+complex-message = *(declaration [s]) complex-body
```

Comment:
This is a preexisting constraint, but it's worth noting that even a complex message is prohibited to have any trailing whitespace by the grammar—including a terminal newline that would generally be expected if represented as the contents of a text file.

Reply:
I agree that it _should_ be supported, but neither the current `complex-message = "{{" [s] *(declaration [s]) body [s] "}}"` nor this `complex-message = *(declaration [s]) complex-body` production accommodate that.

Reply:
Added a `[s]` to the end of `complex-message` to fix this.